docs/technical_reference/unit_models/treatment_models/med_md_semibatch.rst

```diff
@@ -0,0 +1,156 @@
+Low Temperature Multi-effect Distillation - Vacuum Air-gapped Membrane Distillation (LTMED - VAGMD) Semibatch model
+===================================================================================================================
+
+This unit model simulates the semibatch operation of a LTMED - VAGMD system under IDAES multiperiod framework, and
+   * each period in the processing phase contains a steady-state flowsheet as depicted in Fig. 1   
+   * each flowsheet incorporates one LT-MED surrogate model and one VAGMD base model from WaterTAP-REFLO, one Mixer and two Separator models from IDAES
+   * the system performance is evaluated based on a single Aquastill MD module which all inputs are associated to
+   * the system can be scaled to any capacity in the cost evaluation
+
+.. figure:: ../../../_static/unit_models/med_md_semibatch_schematic.png
+    :width: 800
+    :align: center
+
+    Figure 1. Diagram of a MED-MD semibatch system.
```

Comment:
1. The only reason is to create a state block for the remained liquid in the mixer, so that the mixer model can recognize 3 inlet flows (MED brine, MD brine and remained liquid). In another words, the mixer model takes everything as a flow "physically" going into it and generate a mixed flow "physically" going out. So in a multiperiod structure, where part of the mixed flow remains in the mixer, we need to create a "physical" inlet flow as a representative of the remained liquid, and a virtual separator is the most straight-forward way of creating a state block of this flow, I think.

2. For t1, the feed to MD inlet is assumed to have the same properties of the MED brine, because no MD brine has been recirculated to the Mixer at the beginning of the process. I modified the schematic so that it's a physical arrow going from t1 mixer to t2 MD module instead of a dashed line, which could be misleading.


Also I added a _**'Boundary conditions'**_ section in the documentation to describe these detail assumptions at the beginning and the end of the process.